silta/silta.yml

```diff
@@ -7,3 +7,23 @@
 # Example: enable private files for Drupal.
 #privateFiles:
```

Comment:
This example needs to be removed, or at least updated. There's an example in values.yaml, so we might not need anything here after all. This also needs to be updated in the drupal-project repo.